### PR TITLE
Describe fix for #69

### PR DESCRIPTION
**Pull request by in0rdr**
This describes a workaround for #69.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,6 +52,16 @@ This is a text based diary, inspired by [khal](https://github.com/pimutils/khal)
 
 Pre-built packages can be installed from the [Open Build Service (OBS)](https://software.opensuse.org//download.html?project=home%3Ain0rdr&package=diary)
 
+### Arch Linux OBS Repository Bug
+
+The Arch Linux repository on the OBS is broken, see [#69](https://github.com/in0rdr/diary/issues/69) and [#9953](https://github.com/openSUSE/open-build-service/issues/9953).
+
+To use the repository on Arch Linux, use `downloadcontent.opensuse.org` instead of the [suggested](https://software.opensuse.org/download.html?project=home%3Ain0rdr&package=diary) name `download.opensuse.org` in the `/etc/pacman.conf`:
+```
+[home_in0rdr_Arch]
```
Comment (bovine3dom) [inline README.md:61]:
```suggestion
[home_in0rdr_Arch]
SigLevel = Never
```

Without it: 

```
[I] ~ $ sudo pacman -Syy diary
error: home_in0rdr_Arch: key "54A9D1F7050C44ED" is unknown
:: Import PGP key 54A9D1F7050C44ED? [Y/n] y
error: key "54A9D1F7050C44ED" could not be looked up remotely
:: Synchronising package databases...
 home_in0rdr_Arch                                                                        934.0   B  0.00   B/s 00:00 [######################################################################] 100%
 home_in0rdr_Arch.sig                                                                    280.0   B  0.00   B/s 00:00 [######################################################################] 100%
error: home_in0rdr_Arch: key "54A9D1F7050C44ED" is unknown
:: Import PGP key 54A9D1F7050C44ED? [Y/n] y
error: key "54A9D1F7050C44ED" could not be looked up remotely
error: failed to update home_in0rdr_Arch (invalid or corrupted database (PGP signature))
 core                                                                                    131.1 KiB  1409 KiB/s 00:00 [######################################################################] 100%
 extra                                                                                  1649.2 KiB  15.1 MiB/s 00:00 [######################################################################] 100%
 community                                                                                 5.4 MiB  56.5 MiB/s 00:00 [######################################################################] 100%
 multilib                                                                                151.5 KiB  49.3 MiB/s 00:00 [######################################################################] 100%
error: failed to synchronize all databases
```

**Review by bovine3dom — APPROVED** [review]
LGTM - works fine : )

File: README.md (hunk 1 above)
Comment (in0rdr) [reply README.md:61]: @bovine3dom did you try importing the key from OBS like explained in the [install instructions](https://software.opensuse.org/download.html?project=home%3Ain0rdr&package=diary)?

#### Then run the following as root

```
key=$(curl -fsSL https://download.opensuse.org/repositories/home:in0rdr/Arch/$(uname -m)/home_in0rdr_Arch.key)
fingerprint=$(gpg --quiet --with-colons --import-options show-only --import --fingerprint <<< "${key}" | awk -F: '$1 == "fpr" { print $10 }')

pacman-key --init
pacman-key --add - <<< "${key}"
pacman-key --lsign-key "${fingerprint}"
```

The key should actually be readily available:
https://keyserver.ubuntu.com/pks/lookup?search=0x54A9D1F7050C44ED&fingerprint=on&op=index

Comment (bovine3dom) [reply README.md:61]: No, I didn't try that. I think the instructions might have changed? I'm sure it said SigLevel never previously.

I'll try to remember to try again later today.

Comment (bovine3dom) [reply README.md:61]: Yeah, it works fine with those instructions.

**Pull request merged**